node/src/config_files/p2p.rs

```diff
@@ -64,14 +67,14 @@ pub struct P2pConfigFile {
     pub max_inbound_connections: Option<usize>,
     /// The score threshold after which a peer is banned.
     pub ban_threshold: Option<u32>,
-    /// Duration of bans in seconds.
-    pub ban_duration: Option<u64>,
-    /// The outbound connection timeout value in seconds.
-    pub outbound_connection_timeout: Option<NonZeroU64>,
+    /// Duration of bans.
+    pub ban_duration: Option<RpcDuration>,
```

Comment:
Same discussion applies here about units of time. No units of time should be done in Rpc units. We use seconds or milliseconds (whatever is correct and consistent), and then convert to something sensible based on types.

This is the evil part of derive in Rust... now we have floating point numbers in our configuration without even noticing.